tools/st2ctl

```diff
@@ -231,6 +231,13 @@ function getpids(){
       done
     else
       echo "${COM} is not running"
+      if [ "$COM" == "st2sensorcontainer" ] || [ "$COM" == "sensor_container" ]
+      then
+        echo "Sensor container did not detect valid sensors to run."
+        echo "Please install sensors or check sensors configuration for existing ones."
+        echo "See http://docs.stackstorm.com/sensors.html."
+        echo "Use 'st2ctl restart-component st2sensorcontainer' to restart after you fix the problems."
```

Comment:
> Sensor container service was unable to load any sensors to load in the `base_pack` directory (or whatever path this needs to be). This may be caused due to not having any sensors installed, or the sensors that are installed are misconfigured. Take a moment and review http://docs.stackstorm.com/sensors.html for more information. Once resolved, you can simply restart the service with the `st2ctl restart-component st2sensorcontainer` command.